### PR TITLE
Added URL field

### PR DESCRIPTION
**Pull request by Tomicapo**
This is for compatibility with windows servers,
It now saves the upload's URL when you configure the "url" parameter in
the configuration.

### CODE REVIEW

File: src/Model/Behavior/UploadableBehavior.php [hunk 1]
```diff
@@ -38,6 +38,7 @@ class UploadableBehavior extends Behavior
     protected $_defaultConfig = [
         'defaultFieldConfig' => [
             'fields' => [
+                'URL' => false,
```
Comment (bobmulder) [inline src/Model/Behavior/UploadableBehavior.php]: would be cleaner with lowercases `url`?

File: src/Model/Behavior/UploadableBehavior.php [hunk 2]
```diff
@@ -400,6 +404,21 @@ protected function _getPath($entity, $field, $options = [])
...
+     * Returns the URL of the given field.
+     *
+     * @param \Cake\ORM\Entity $entity Entity to check on.
+     * @param string $field Field to check on.
+     * @return string
+     */
+    protected function _getUrl($entity, $field)
+    {
+        $path = '/' . $this->_getPath($entity, $field, ['root' => false, 'file' => true]);
+        return str_replace('\\', '/', $path);
```
Comment (bobmulder) [inline src/Model/Behavior/UploadableBehavior.php]: Just a suggestion: instead of `\\` you could use `DS` or not?

**Comment by bobmulder** [comment]
Great work! Code looks very clean! Just would like to see the updated tests. Just waiting for Travis CI

**Comment by Tomicapo** [comment]
Just added your suggestions. Let's wait for the tests to run.

**Comment by bobmulder** [comment]
I think you did a great job @Tomicapo. For sure I will check it out myself and then will merge the PR.

**Comment by bobmulder** [comment]
@Tomicapo Docs are still missing. I'll fix them.

**Pull request merged**